types/react-countup/index.d.ts

```diff
@@ -0,0 +1,29 @@
+// Type definitions for react-countup 3.0.2
+// Project: https://github.com/glennreyes/react-countup
+// Definitions by: Ivan Jiang <https://github.com/iplus26>
+// Definitions: https://github.com/DefinitelyTyped/DefinitelyTyped
+// TypeScript Version: 2.5.2
+
+declare module "react-countup" {
```

Comment:
No need to create an ambient external module. The wrapper should be removed entirely.